x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/test/TimeWarpedWatcher.java

```diff
@@ -36,8 +35,7 @@
 
     public TimeWarpedWatcher(final Settings settings, final Path configPath) throws Exception {
         super(settings, configPath);
-        Logger logger = Loggers.getLogger(TimeWarpedWatcher.class, settings);
-        logger.info("using time warped watchers plugin");
+        LogManager.getLogger(TimeWarpedWatcher.class).info("using time warped watchers plugin");
```

Comment:
I don't know how often this class is created, but maybe it makes sense to store a reference to the logger for later use if this class is instantiated more often.